tests/framework/CodeInterfaceTests/raven_running_raven_internal_models/test_rom_trainer.xml

```diff
@@ -160,7 +192,7 @@
     </PointSet>
     <PointSet name="outputMontecarloRom">
       <Input>DeltaTimeScramToAux,DG1recoveryTime</Input>
-      <Output>CladTempThreshold</Output>
+      <!--Output>CladTempThreshold</Output-->
```

Comment:
rm?

Reply:
This is part of the unfinished fixing, as mentioned in the description.  It is ongoing work.

Reply:
ok